package.json

```diff
@@ -27,7 +27,7 @@
     "format": "prettier ./ --write",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "pre-publish": "webpack --mode=production",
+    "pre-publish": "npm version patch && webpack --mode=production",
```

Comment:
@yi-xuan-97 we don't need to change the version number, because we are using sementic-release, it automatically update the version number for every merge: https://github.com/Greenstand/treetracker-web-map-core/blob/main/.releaserc